extensions/license/main.ts

```diff
@@ -0,0 +1,14 @@
+import { LensMainExtension } from "@k8slens/extensions";
+import { shell } from "electron";
+
+export default class LicenseLensMainExtension extends LensMainExtension {
+  appMenus = [
+    {
+      parentId: "help",
+      label: "License",
+      async click() {
+        shell.openExternal("https://k8slens.dev/licenses/eula.md");
```

Comment:
Would be safer if we expose `openExternal` (or whole `shell`) via extension-api (maybe via `Util` namespace?).